VideoWeb/VideoWeb.AcceptanceTests/Helpers/Users.cs

```diff
@@ -29,6 +29,16 @@ public static UserDto GetUserFromDisplayName(List<UserDto> users, string display
             var usersList = users.Select(x => x.DisplayName).Aggregate("", (current, name) => current + name + ",");
             throw new InvalidOperationException($"No user with display name '{displayName}' found in the list: '{usersList}'");
         }
+        public static UserDto GetUserFromUserType(List<UserDto> users, string userType)
+        {
+            if (users.Any(x => x.UserType.ToString().ToLower().Equals(userType.ToLower())))
+            {
+                return users.First(x => x.UserType.ToString().ToLower().Equals(userType.ToLower()));
+            }
+
+            var usersList = users.Select(x => x.UserType).Aggregate("", (current, UserType) => current + userType + ",");
+            throw new InvalidOperationException($"No user with display name '{userType}' found in the list: '{usersList}'");
```

Comment:
Looks like there is a typo in the exception message ;) It should be  $"No user with user type '{userType}' found in the list: '{usersList}'"

Reply:
Thanks, I fixed it,